bedrock/firefox/templates/firefox/concerts.html

```diff
@@ -2,48 +2,189 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
-{% extends "firefox/base-pebbles.html" %}
+{% extends "firefox/base-protocol.html" %}
 
-{% block page_title %}Give email, watch concert{% endblock %}
+{% block page_title_prefix %}Firefox Concert Series - {% endblock %}
+{% block page_title %}Watch free live concerts in Firefox{% endblock %}
+{% block page_desc %}Here’s to you. Here’s to live music. Here’s to watching free, from all kinds of artists, right from Firefox. Thanks for being part of our community. Firefox fights for you.{% endblock %}
 
 {% block page_css %}
-  {{ css_bundle('firefox_concerts') }}
+  {{ css_bundle('firefox_concert_series') }}
 {% endblock %}
 
+{% block body_class %}firefox-concerts{% endblock %}
+
 {% block content %}
 <main role="main">
-  <div class="content">
-    <div class="conditional-content verified-content">
-      <p>You have been verified!</p>
+  <div class="concert-content">
+    <header class="mzp-c-hero mzp-t-firefox mzp-t-product-firefox mzp-t-dark">
+      <div class="mzp-l-content">
+        <div class="mzp-c-hero-body">
+          <h1 class="mzp-c-hero-title">Here’s to you.<br> Here’s to live music.</h1>
+          <div class="mzp-c-hero-desc">
+            <p>Here’s to watching it free, from all kinds of artists, right from Firefox.</p>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <div class="artist-featured mzp-l-content l-content-narrow">
+      <section class="artist-card mzp-c-card-feature mzp-has-aspect-16-9 mzp-t-firefox mzp-t-dark">
+        <div class="mzp-c-card-feature-media-wrapper">
+          <div class="mzp-c-card-feature-media">
+            <img src="{{ static('img/firefox/concerts/phosphorescent.jpg') }}" alt="">
+          </div>
+        </div>
+        <div class="mzp-c-card-feature-content">
+          <div class="mzp-c-card-feature-content-container">
+          {% if switch('firefox-concert-one-live') %}
+            <p class="mzp-c-card-tag live">Live</p>
+          {% elif switch('firefox-concert-one-over') %}
+            <p class="mzp-c-card-tag ended">Ended</p>
+          {% else %}
+            <p class="mzp-c-card-tag countdown" id="countdown-one">Starts in <span class="time days"></span> days, <span class="time hours"></span> hours</p>
+          {% endif %}
+            <p class="date"><time datetime="2018-12-11T21:30:00-0500">December 11 at 9:30pm Eastern</time></p>
+            <h2 class="mzp-c-card-feature-title">Phosphorescent</h2>
+            <p class="mzp-c-card-feature-desc">Live from DC, get a front row seat for one of the last shows on Phosphorescent’s <cite lang="fr">C’est La Vie</cite> tour.</p>
+
+          {% if switch('firefox-concert-one-live') %}
+            <p class="stream-link"><a href="#TBD" class="mzp-c-button mzp-t-primary mzp-t-dark">Watch Now</a></p>
```

Comment:
This button doesn't do anything.

Of course, I wouldn't be seeing it if I wasn't in dev.

Reply:
Yeah, we won't get the stream URL from LiveNation until later so we already know there will be at least one update to this page before showtime.